pkg/assembler/backends/testing/backend.go

```diff
@@ -128,6 +133,13 @@ func noMatch(filter *string, value string) bool {
 	return false
 }
 
+func noMatchLowerCase(filter *string, value string) bool {
+	if filter != nil {
+		return value != strings.ToLower(*filter)
+	}
+	return false
+}
+
```

Comment:
Should we instead canonicalize before trying to ingest? This way, we have functions to convert to lower case, remove `-`, etc, and then the match functions can stay as they are